### PR TITLE
Updated e2e tests to load configuration from a file

### PR DESCRIPTION
**Pull request by clintonb**
LEARNER-509

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/learner @edx/helio please review

File: e2e/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Packages required to run e2e tests
-bok-choy==0.6.2
+bok-choy==0.7.0
 ddt==1.1.1
 django-nose==1.4.4
 edx-rest-api-client==1.7.1
 nose-ignore-docstring==0.2
-selenium>=3.0.2
+python-dotenv==0.6.4
+selenium>=3.4.2
```
Comment (rlucioni) [inline e2e/requirements.txt]: Please pin this! I thought we'd learned our lesson by now.

File: e2e/config.py [hunk 2]
```diff
@@ -1,7 +1,14 @@
 import os
+from os.path import dirname, join
+
+from dotenv import load_dotenv
 
 from e2e.utils import str2bool
 
+# Load environment variables from an external file
+dotenv_path = os.environ.get('DOTENV_PATH', join(dirname(__file__), '.env'))
```
Comment (rlucioni) [inline e2e/config.py]: This is cool, but what's wrong with the more conventional YAML loading approach? Introducing a dependency for this seems like overkill.

Comment (clintonb) [reply e2e/config.py]: Why write more code when someone else has already done the work?

File: e2e/requirements.txt (hunk 1 above)
Comment (clintonb) [reply e2e/requirements.txt]: (okay)

File: e2e/config.py (hunk 2 above)
Comment (rlucioni) [reply e2e/config.py]: Well, this dotenv business is now one more thing I need to know about when working in this system. Granted, it's pretty straightforward and I don't expect it to be a problem. I just would have gone with the YAML approach we see elsewhere and been done with it. ¯\\_(ツ)_/¯

**Review by rlucioni — APPROVED** [review]

**Pull request merged**